examples/simple_script/BUILD

```diff
@@ -13,6 +13,7 @@ ruby_binary(
     deps = [
         "//lib:foo",
         "@bundle//:awesome_print",
+        "@bundle//:rails",
```

Comment:
Is this change related to rubocop?

Reply:
this was a test branch, which I have cleaned up on https://github.com/bazelruby/rules_ruby/pull/38